exports/exports.go

```diff
@@ -326,7 +326,7 @@ func APIExportToDBExport(apiPayload ExportPayload) (*models.ExportPayload, error
 	}
 
 	switch apiPayload.Status {
-	case "complete":
+	case "completed_at":
```

Comment:
This references the status of export being `complete`, which is different from the time it was completed `completed_at`.